### ISSUE TITLE
Cannot use parallelStream with SecurityManager

### ISSUE THREAD

**Issue by kamcio96** [SpigotMC/BungeeCord#1732]
```
16:16:32 [WARNING] Plugin performed restricted action, please inform them to use proper API methods: Illegal thread group access
java.security.AccessControlException: Plugin violation: Illegal thread group access
    at net.md_5.bungee.BungeeSecurityManager.checkRestricted(BungeeSecurityManager.java:39)
    at net.md_5.bungee.BungeeSecurityManager.checkAccess(BungeeSecurityManager.java:62)
    at java.lang.ThreadGroup.checkAccess(Unknown Source)
    at java.lang.Thread.init(Unknown Source)
    at java.lang.Thread.init(Unknown Source)
    at java.lang.Thread.<init>(Unknown Source)
    at java.util.concurrent.ForkJoinWorkerThread.<init>(Unknown Source)
    at java.util.concurrent.ForkJoinWorkerThread$InnocuousForkJoinWorkerThread.<init>(Unknown Source)
    at java.util.concurrent.ForkJoinPool$InnocuousForkJoinWorkerThreadFactory$1.run(Unknown Source)
    at java.util.concurrent.ForkJoinPool$InnocuousForkJoinWorkerThreadFactory$1.run(Unknown Source)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.util.concurrent.ForkJoinPool$InnocuousForkJoinWorkerThreadFactory.newThread(Unknown Source)
    at java.util.concurrent.ForkJoinPool.createWorker(Unknown Source)
    at java.util.concurrent.ForkJoinPool.tryAddWorker(Unknown Source)
    at java.util.concurrent.ForkJoinPool.signalWork(Unknown Source)
    at java.util.concurrent.ForkJoinPool.externalPush(Unknown Source)
    at java.util.concurrent.ForkJoinTask.fork(Unknown Source)
    at java.util.stream.AbstractShortCircuitTask.compute(Unknown Source)
    at java.util.concurrent.CountedCompleter.exec(Unknown Source)
    at java.util.concurrent.ForkJoinTask.doExec(Unknown Source)
    at java.util.concurrent.ForkJoinTask.doInvoke(Unknown Source)
    at java.util.concurrent.ForkJoinTask.invoke(Unknown Source)
    at java.util.stream.FindOps$FindOp.evaluateParallel(Unknown Source)
    at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
    at java.util.stream.ReferencePipeline.findFirst(Unknown Source)
    ...
```

**Comment by Janmm14** [comment]
Current bungee builds do not enforce this role and only inform about this issue. Chwck the bungee build and look in the commit history. The do-not-enforce state is already some weeks or months old.

**Comment by kamcio96** [comment]
I know, but it is still issue

**Comment by Janmm14** [comment]
Actually thats intended behaviour in that case, I think.

**Comment by md-5** [comment]
Fixed in 1.9
